zk_toolbox/crates/zk_inception/src/commands/chain/args/init.rs

```diff
@@ -28,6 +28,8 @@ pub struct InitArgs {
     pub deploy_paymaster: Option<bool>,
     #[clap(long, help = MSG_L1_RPC_URL_HELP)]
     pub l1_rpc_url: Option<String>,
+    #[clap(long)]
+    pub copy_configs: Option<bool>,
```

Comment:
i can't imagine why, i won't it to be copied

Reply:
Well, if they were already copied with prover init. Although, not a big deal I believe, I can remove it